<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddDataForm</class>
    <widget class="QDialog" name="AddDataForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>411</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Добавить работу</string>
        </property>
        <property name="styleSheet">
            <string>
                QDialog {
                background-color: rgb(88, 88, 88);
                }

                QLabel {
                color: rgb(255, 255, 255);
                }

                QLineEdit {
                background-color: #f0f0f0;
                border: 1px solid #ccc;
                border-radius: 5px;
                padding: 8px;
                font-size: 14px;
                }

                QPushButton {
                background-color: #4CAF50;
                color: white;
                border-radius: 5px;
                font-size: 16px;
                padding: 8px 16px;
                }

                QPushButton:hover {
                background-color: #45a049;
                }

                QPushButton:pressed {
                background-color: #0e6b0e;
                }

                QPushButton#btnSelectFile {
                background-color: #2da9d2;
                color: white;
                border-radius: 5px;
                font-size: 16px;
                padding: 8px 16px;
                }

                QPushButton#btnSelectFile:hover {
                background-color: #258eb1;
                }

                QPushButton#btnSelectFile:pressed {
                background-color: #1b6a85;
                }
                QPushButton#btnCancel {
                background-color: #FF0000;
                }

                QPushButton#btnCancel:hover {
                background-color: #CC0000;
                }

                QPushButton#btnCancel:pressed {
                background-color: #990000;
                }
            </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>Название работы:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="lineEditTitle"/>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Тип работы:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="lineEditType"/>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="label_3">
                            <property name="text">
                                <string>Автор (Фамилия Имя):</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLineEdit" name="lineEditAuthor"/>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="label_7">
                            <property name="text">
                                <string>Группа:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QLineEdit" name="lineEditGroup">
                            <property name="text">
                                <string>М3О-310Б-21</string>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="0">
                        <widget class="QLabel" name="label_4">
                            <property name="text">
                                <string>Дисциплина:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="1">
                        <widget class="QLineEdit" name="lineEditDiscipline"/>
                    </item>
                    <item row="5" column="0">
                        <widget class="QLabel" name="label_5">
                            <property name="text">
                                <string>Год публикации:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="5" column="1">
                        <widget class="QLineEdit" name="lineEditYear"/>
                    </item>
                    <item row="7" column="0">
                        <widget class="QLabel" name="label_6">
                            <property name="text">
                                <string>Выбрать файл работы:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="7" column="1">
                        <widget class="QPushButton" name="btnSelectFile">
                            <property name="text">
                                <string>Выбрать файл</string>
                            </property>
                        </widget>
                    </item>
                    <item row="6" column="0">
                        <widget class="QLabel" name="label_8">
                            <property name="text">
                                <string>Место хранения работы:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="6" column="1">
                        <widget class="QLineEdit" name="lineEditStorage"/>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="btnAdd">
                            <property name="text">
                                <string>Добавить</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btnCancel">
                            <property name="text">
                                <string>Отмена</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
